src/components/Options/TimelineOptions.js

```diff
@@ -0,0 +1,16 @@
+const IsEnabledOptions = [
+  { value: "on", label: "ON" },
+  { value: "off", label: "OFF" },
+];
+
+const HasSyncedOptions = [
+  { value: "True", label: "TRUE" },
+  { value: "False", label: "FALSE" },
+];
+
+const InfiniteSessionsOptions = [
+  { value: "True", label: "TRUE" },
```

Comment:
Why is `value` a string and not boolean?

Reply:
Updated to boolean